### ISSUE TITLE
gulp-modernizr uses old modernizr metadata api

### ISSUE THREAD

**Issue by mahnunchik** [doctyper/gulp-modernizr#22]
Hi,

Since this commit https://github.com/Modernizr/Modernizr/commit/b8290c6f0bb57a47127d614725c40cc25ca4968d Modernizr exposes another metadata api.

In its current state `gulp-modernizr` is completely broken.

**Comment by mahnunchik** [comment]
This issue is more addressed to the https://github.com/doctyper/customizr

**Comment by adammockor** [comment]
This log is probably related.

```
16:33:31] Starting 'modernizr'...

Error: Cannot find module 'file'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/adam/Praca/temp/1.2.0/node_modules/gulp-modernizr/node_modules/customizr/node_modules/modernizr/lib/metadata.js:2:12)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/adam/Praca/temp/1.2.0/node_modules/gulp-modernizr/node_modules/customizr/node_modules/modernizr/lib/cli.js:5:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

**Comment by mahnunchik** [comment]
@adammockor it is more related to @Modernizr itself, it has `file` only in dev dependancies.

 `gulp-responsive` uses metadata API of Modernizr which requires `file` module.

**Comment by adammockor** [comment]
So, it's not related to your issue? I am confused.

**Comment by mahnunchik** [comment]
@doctyper is in https://github.com/Modernizr org, so I think he can fix both repositories.

**Comment by doctyper** [comment]
This issue is being worked on here: https://github.com/doctyper/customizr/tree/feature/new-build-system
There is an open ticket in doctyper/customizr#9. Please track your issues there.

**Comment by doctyper** [comment]
Fixed in doctyper/customizr@b037e2269492ba83dcc561f3fadf6d6b9305922a. [Passing all tests](https://travis-ci.org/doctyper/customizr/builds/50235214).
